### PR TITLE
Added Player::ResetInventory(IEnumerable<string>)

### PR DESCRIPTION
**Pull request by Monaldcry7788**
Added Player::ResetInventory(IEnumerable<string>), reduced nesting in ResetInventory methods, and aligned style for foreach

### CODE REVIEW

File: Exiled.API/Features/Player.cs [hunk 1]
```diff
@@ -1837,12 +1837,30 @@ public void ResetInventory(IEnumerable<Item> newItems)
...
+            if (!newItems.Any())
+                return;
+
+            foreach (string item in newItems)
             {
-                foreach (Item item in newItems)
-                {
-                    AddItem(item.Base == null ? new Item(item.Type) : item);
-                }
+                if (!Enum.TryParse(item, out ItemType itemType))
```
Comment (Jesus-QC) [inline Exiled.API/Features/Player.cs:1860]: 🤨

Comment (Monaldcry7788) [reply Exiled.API/Features/Player.cs:1860]: What?

**Comment by Jesus-QC** [comment]
Is it necessary? Why dont you just create a list of itemtypes.

**Comment by Monaldcry7788** [comment]
it's just an addition that someone might need.

**Comment by Jesus-QC** [comment]
Well, I dont think it shoukd be approved, letting the rest of the team decide.

**Comment by joker-119** [comment]
I don't really see the point of this, the only reason we have string related inventory stuff is because of Customitems, but those are in the CustomItem project for exiled already. You shouldn't really be using a string (or strings) to represent items in a player's inventory.

**Comment by Monaldcry7788** [comment]
Mh.. ok. I will close it

**Pull request closed without merging**